MapboxSearch/offline/api/api-metalava.txt

```diff
@@ -94,11 +94,11 @@ package com.mapbox.search.offline {
     method public void addOnIndexChangeListener(java.util.concurrent.Executor executor, com.mapbox.search.offline.OfflineSearchEngine.OnIndexChangeListener listener);
     method public default void addOnIndexChangeListener(com.mapbox.search.offline.OfflineSearchEngine.OnIndexChangeListener listener);
     method public default static com.mapbox.search.offline.OfflineSearchEngine create(com.mapbox.search.offline.OfflineSearchEngineSettings settings);
-    method public default static com.mapbox.common.TilesetDescriptor createPlacesTilesetDescriptor(String dataset = "test-dataset", String version = "");
-    method public default static final com.mapbox.common.TilesetDescriptor createPlacesTilesetDescriptor(String dataset = "test-dataset");
+    method public default static com.mapbox.common.TilesetDescriptor createPlacesTilesetDescriptor(String dataset = "mbx-main", String version = "");
```

Comment:
could we move `mbx-main` to a global constant to make available switch dataset in single click

Reply:
This is an autogenerated file that tracks public API. We can't move any constants out of this.